odyssey-tmp/odyssey-device-deploy.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/zsh
```

Comment:
What's the point of adding this to the repo? It's meant to work online and always fetch the latest script.